src/app-header.c

```diff
@@ -149,16 +153,19 @@ void app_header_init(AppHeader* ah)
 	gtk_button_set_image(GTK_BUTTON(ah->btn_sync), ah->icon_sync);
 	g_signal_connect_swapped(ah->btn_sync, "clicked", (GCallback) &app_header_sync, ah);
 	gtk_header_bar_pack_end(GTK_HEADER_BAR(ah), ah->btn_sync);
+  gtk_widget_set_tooltip_text(ah->btn_sync, "Synchronize");
 
 	ah->btn_delete = gtk_button_new();
 	gtk_button_set_image(GTK_BUTTON(ah->btn_delete), gtk_image_new_from_icon_name("edit-delete-symbolic", GTK_ICON_SIZE_SMALL_TOOLBAR));
 	g_signal_connect_swapped(ah->btn_delete, "clicked", (GCallback) &app_header_delete, ah);
 	gtk_header_bar_pack_end(GTK_HEADER_BAR(ah), ah->btn_delete);
+  gtk_widget_set_tooltip_text(ah->btn_delete, "Delete Selected");
```

Comment:
The event takes up the full screen when this button is shown, so it might not be understood as "selected" in the ususal sense. I'd just say "Delete Event"